dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/Ds3GetJob_Test.java

```diff
@@ -80,14 +110,27 @@ public static void setConnection() {
             final DeepStorageBrowserTaskProgressView<Ds3JobTask> taskProgressView = new DeepStorageBrowserTaskProgressView<>();
             Mockito.when(deepStorageBrowserPresenter.getJobProgressView()).thenReturn(taskProgressView);
             try {
-                ds3GetJob = new Ds3GetJob(listTreeTable, path, ds3Client, Priority.URGENT.toString(), 5, JobInterruptionStore.loadJobIds(), ds3Common);
+                ds3GetJob = new Ds3GetJob(listTreeTable, path, ds3Client, Priority.URGENT.toString(), 5, JobInterruptionStore.loadJobIds(), deepStorageBrowserPresenter, session, resourceBundle, null);
                 taskProgressView.getTasks().add(ds3GetJob);
             } catch (final IOException io) {
                 io.printStackTrace();
             }
+
+            try {
+                envDataPolicyId = IntegrationHelpers.setupDataPolicy(TEST_ENV_NAME, false, ChecksumType.Type.MD5, client);
+                envStorageIds = IntegrationHelpers.setup(TEST_ENV_NAME, envDataPolicyId, client);
+                HELPERS.ensureBucketExists(DS3GETJOB_TEST_BUCKET_NAME, envDataPolicyId);
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.